testDataSetup/mongo/mongo.go

```diff
@@ -339,6 +391,20 @@ func GetInstance(database, collection, key, value string) (Instance, error) {
 	return instance, nil
 }
 
+// CountDimensionOptions retrieves a count of the number of dimension options exist for an instance in mongo
+func CountDimensionOptions(database, collection, key, value string) (int, error) {
+	s := session.Copy()
+	defer s.Close()
+
+	var count int
+	count, err := s.DB(database).C(collection).Find(bson.M{key: value}).Count()
```

Comment:
Can this be changed to `return s.DB(database).C(collection).Find(bson.M{key: value}).Count()`, as err will either be nil or an error type.